docs/ai/javascript.rst

```diff
@@ -0,0 +1,167 @@
+.. _ref_ai_javascript:
+
+==========
+JavaScript
+==========
+
+:edb-alt-title: EdgeDB AI's JavaScript package
+
+``@edgedb/ai`` offers a convenient wrapper around ``ext::ai``. Install it with
+npm or via your package manager of choice:
+
+.. code-block:: bash
+
+    $ npm install @edgedb/ai # or
+    $ yarn add @edgedb/ai # or
+    $ pnpm add @edgedb/ai
+
+
+Usage
+=====
+
+Start by importing ``createClient`` from ``edgedb`` and ``createAI`` from
+``@edgedb/ai``:
+
+.. code-block:: typescript
+
+    import { createClient } from "edgedb";
+    import { createAI } from "@edgedb/ai";
+
+Create an EdgeDB client. Create an instance of the AI client by passing in the
+EdgeDB client and any options for the AI provider (like the text generation
+model):
+
+.. code-block:: typescript
+
+    const client = createClient();
+
+    const gpt4Ai = createAI(client, {
+      model: "gpt-4-turbo-preview",
+    });
```

Comment:
Should we link to somewhere where we explain what models user can use?

Reply:
I'm glad you caught this one because I didn't have these listed in the reference at all. (They were listed only on the overview page.)

I listed them, linked to them from here, and took all of the model lists out of the overview page and instead linked to the lists in the reference.